__docs__/wonder-blocks-form/text-area-testing-snapshots.stories.tsx

```diff
@@ -73,6 +78,41 @@ export const StateSheetStory: Story = {
     },
 };
 
+/**
+ * What the component looks like at rest with different combinations of props.
+ */
+export const Combinations: Story = {
+    render: (args) => {
+        return (
+            <AllVariants
```

Comment:
Creating a new story for testing the component with a value and placeholder in different states. We don't include this in the StateSheet because it's unnecessary to see these things with all the different pseudo states (it's enough to show these cases at rest!)